modules/calendar/event.py

```diff
@@ -4,78 +4,122 @@
 
 
 class Event:
-	def __init__(self, details: Dict[str, str], tz: str = None):
-		self.details = details
-		self.tz = tz or self.details["start"].get("timeZone", None)
-
+	"""Event object to store data about a Google Calendar event"""
+
+	def __init__(
+		self,
+		event_id: str,
+		link: str,
+		title: str,
+		location: str,
+		description: str,
+		all_day: bool,
+		start: datetime,
+		end: datetime,
+		timezone: str,
+	):
+		self.__id = event_id
+		self.__link = link
+		self.__title = title
+		self.__location = location
+		self.__description = description
+		self.__all_day = all_day
+		self.__start = start
+		self.__end = end
+		self.__timezone = timezone
+
+	@property
 	def id(self) -> str:
```

Comment:
```suggestion
	def event_id(self) -> str:
```
better to not use same names as built in functions

Reply:
Objects do not have an `id` property unless it is specifically declared, so this does not cause any confusion.

Reply:
Even the Discord API uses id properties in classes